.github/workflows/docker-build-and-push.yaml

```diff
@@ -48,6 +51,9 @@ jobs:
         run: |
           export TAG=`echo $GITHUB_REF | awk -F/ '{print $NF}'`
           echo $TAG
-          docker push "$REGISTRY_HOSTNAME"/"$IMAGE":"$TAG"
-          docker tag "$REGISTRY_HOSTNAME"/"$IMAGE":"$TAG" "$REGISTRY_HOSTNAME"/"$IMAGE":latest
-          docker push "$REGISTRY_HOSTNAME"/"$IMAGE":latest
+          docker push "$ARTIFACT_REGISTRY_HOSTNAME"/"$IMAGE":"$TAG"
```

Comment:
docker push "$ARTIFACT_REGISTRY_HOSTNAME"/"$IMAGE":"$TAG"
          docker tag "$ARTIFACT_REGISTRY_HOSTNAME"/"$IMAGE":"$TAG" "$ARTIFACT_REGISTRY_HOSTNAME"/"$IMAGE":latest
          docker push "$ARTIFACT_REGISTRY_HOSTNAME"/"$IMAGE":latest



do we still need these ?

Reply:
I'm afraid that someone might already have sent those links around, so I'm a little reluctant to remove them right away.

I'll open a separate PR for it right away, so we can remove it quickly if we want to. (Note: A new image is only pushed if we do a release. We don't push a new version for each commit that is done.)